adafruit_tlv493d.py

```diff
@@ -175,10 +175,9 @@ def magnetic(self) -> Tuple[float, float, float]:
         )
 
     # pylint: disable=no-self-use
-    def _unpack_and_scale(
-        self, top: int, bottom: int
-    ) -> float:
+    def _unpack_and_scale(self, top: int, bottom: int) -> float:
```

Comment:
Try:

```python
    def _unpack_and_scale(  # pylint: disable=no-self-use
        self, top: int, bottom: int
    ) -> float:
```

Reply:
well, I swear I tried that before.  whatever changed - had put pylintrc back to it's original state.  Looks like it passed this time!